### DISCUSSION TITLE
Startup Error Box appear

### DISCUSSION THREAD

**Discussion by chnisar515** [comfyanonymous/ComfyUI#5240] [General]
Hello Experts,
Today i have installed ([ComfyUI-UniAnimate-W]) this node and its dependencies after that when i try to restart my ComfyUI it did not start the i manually update all the things from Update folder and run the ComfyUI portable with GPU file then on startup its showing me an error box i have attached the file so please guide me how i can fix this issue. by the way my ComfyUI working good but that error just irritating me.
![Screenshot 2024-10-14 035207](https://github.com/user-attachments/assets/a68669f3-1a6f-420a-a187-5f1d7ea6c3a7)

**Comment by ltdrdata** [comment]
Your issue is caused by installing xformers that is incompatible with pytorch. If xformers is not absolutely necessary, I recommend removing it. 
If it is essential, you should reinstall it with a version that is compatible with your pytorch.

**Comment by chnisar515** [comment]
how i can remove xformers ? and how i can know which xformers version is compatible with my pytorch ?

**Comment by chnisar515** [comment]
i tried this command to uninstall xformers and i got this resutl.

C:\ComfyUI_windows_portable>pip uninstall xformers
WARNING: Skipping xformers as it is not installed.

**Reply by LukeG89** [reply]
To uninstall xformers in your portable version you need to open cmd from `ComfyUI_windows_portable` folder and use this command: 
```
python_embeded\python.exe -m pip uninstall xformers
```

**Reply by chnisar515** [reply]
Thank you for your reply. also can you guide me how i can reinstall xformers right version for pytorch?

**Reply by LukeG89** [reply]
What's your pytorch version?

**Reply by chnisar515** [reply]
i got this error on this point (pytorch version: 2.4.1+cu124) and when i click on OK button on error screen popup then it start to load. below is the cmd output where i can see 2 type of pytorch version like (PyTorch 2.3.1+cu121 with CUDA 1201 (you have 2.4.1+cu124))

C:\ComfyUI_windows_portable>.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build
[START] Security scan
[DONE] Security scan
## ComfyUI-Manager: installing dependencies done.
** ComfyUI startup time: 2024-10-14 15:52:22.978015
** Platform: Windows
** Python version: 3.11.9 (tags/v3.11.9:de54cf5, Apr  2 2024, 10:12:12) [MSC v.1938 64 bit (AMD64)]
** Python executable: C:\ComfyUI_windows_portable\python_embeded\python.exe
** ComfyUI Path: C:\ComfyUI_windows_portable\ComfyUI
** Log path: C:\ComfyUI_windows_portable\comfyui.log

Prestartup times for custom nodes:
   0.0 seconds: C:\ComfyUI_windows_portable\ComfyUI\custom_nodes\rgthree-comfy
   1.7 seconds: C:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager

Total VRAM 12288 MB, total RAM 32607 MB
pytorch version: 2.4.1+cu124
WARNING[XFORMERS]: xFormers can't load C++/CUDA extensions. xFormers was built for:
    PyTorch 2.3.1+cu121 with CUDA 1201 (you have 2.4.1+cu124)
    Python  3.11.9 (you have 3.11.9)
  Please reinstall xformers (see https://github.com/facebookresearch/xformers#installing-xformers)
  Memory-efficient attention, SwiGLU, sparse and more won't be available.
  Set XFORMERS_MORE_DETAILS=1 for more details
C:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\xformers\ops\fmha\flash.py:211: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.
  @torch.library.impl_abstract("xformers_flash::flash_fwd")

**Reply by LukeG89** [reply]
First, you need to uninstall xformers.

Then, you can reinstall it using this command:
```
python_embeded\python.exe -m pip install xformers --extra-index-url https://download.pytorch.org/whl/cu124
```

**Reply by chnisar515** [reply]
Thank you so so so much dear LukeG89. i followed your steps and uninstalled xformers first and then i reinstall it with your provided command and now the error is gone and everything worked perfectly. Thank you again.
